.github/workflows/hive.yaml

```
@@ -0,0 +1,239 @@
+name: Hive - EOF
+on:
+  schedule:
+    - cron: '45 12 * * *'
+  workflow_dispatch:
+    # Note: We're limited to 10 inputs
+    inputs:
+      client:
+        type: string
+        default: '"go-ethereum","reth","nethermind","ethereumjs","besu","erigon"'
+        description: Comma-separated list of clients to test .e.g go-ethereum, besu, reth, nethermind, erigon, ethereumjs
+      simulator:
+        type: string
+        default: '"ethereum/eest/consume-engine","ethereum/eest/consume-rlp"'
+        description: Comma-separated list of simulators to test ethereum/eest/consume-engine, ethereum/eest/consume-rlp
+      hive_version:
+        type: string
+        default: ethereum/hive@master
+        description: GitHub repository and tag for hive (repo@tag)
+      geth_version:
+        type: string
+        default: ethereum/go-ethereum@eof-devnet-0
+        description: GitHub repository and tag for go-ethereum (repo@tag)
+      besu_version:
+        type: string
+        default: hyperledger/besu@main
+        description: GitHub repository and tag for besu (repo@tag)
+      reth_version:
+        type: string
+        default: paradigmxyz/reth@main
+        description: GitHub repository and tag for reth (repo@tag)
+      nethermind_version:
+        type: string
+        default: NethermindEth/nethermind@feature-evm-eof
+        description: GitHub repository and tag for nethermind (repo@tag)
+      erigon_version:
+        type: string
+        default: erigontech/erigon@eof-tests
+        description: GitHub repository and tag for erigon (repo@tag)
+      ethereumjs_version:
+        type: string
+        default: ethereumjs/ethereumjs-monorepo@master
+        description: GitHub repository and tag for ethereumjs (repo@tag)
+
+env:
+  # Proxy
+  GOPROXY: "${{ vars.GOPROXY }}"
+  # Hive action environment variables
+  S3_BUCKET: hive-results
+  S3_PATH: eof-devnet-0
+  S3_PUBLIC_URL: https://hive.ethpandaops.io/eof-devnet-0
+  INSTALL_RCLONE_VERSION: v1.68.2
+  EEST_BUILD_ARG_FIXTURES: https://github.com/ethereum/execution-spec-tests/releases/tag/eip7692%40v2.2.0
```

```suggestion
  EEST_BUILD_ARG_FIXTURES: https://github.com/ethereum/execution-spec-tests/releases/tag/eip7692%40latest
```

What about using `latest` here? I think it should work out of the box.

---

Works for me :)

---

Added as `eip7692@latest`!

---

@spencer-tb, @marioevz  I'd prefer to be explicit and bump this when necessary. This is a dashboard specifically for eof-devnet-1, future releases may not be compatible with clients at devnet-1 specs. When a future fixture release with incompatible specs comes out this will likely cause clients to suddenly start failing tests.

Also, while we can use `latest` for the fixtures, we can't for the `branch`, so whatever branch we specify there will (likely) also become incompatible with the `latest` fixtures at some point in time.

---

https://github.com/ethpandaops/eof-devnets/pull/7
Makes sense to me :)